### DISCUSSION TITLE
bug: Bad links in extension documentation that appears in VSCode

### DISCUSSION THREAD

**Discussion by RossBoylan** [krvajal/vscode-fortran-support#895] [Q&A - deprecated see Fortran Discourse] [answered]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Description

There are invalid links in several spots in VSCode.

The page describing the extension contains links that do not work (yield 404 not found when opened).
Under Useful Tools the icon for fortls points to https://gnikit.github.io/fortls/ (screenshot 1 below)
Further down it says "For more about the Language Server's capabilities please refer to the documentation".  The last word is a link to https://gnikit.github.io/fortls/. (screenshot 2)

Similarly, in the settings for fortls there are references to gnikit.gihub.io that do not work. (screenshot 3)

### Screenshots

![image](https://github.com/fortran-lang/vscode-fortran-support/assets/2514994/e0debd02-1a45-466b-a60d-2694714016ca)
![image](https://github.com/fortran-lang/vscode-fortran-support/assets/2514994/9290e6cf-d36a-4447-8d75-7993647a694b)
![image](https://github.com/fortran-lang/vscode-fortran-support/assets/2514994/d8c74048-4cb2-43d6-b525-4053eb604c62)

### Expected Behaviour

I expected the links would take me to actual documentation

### Version of Modern Fortran

3.2.0

### Version of Visual Studio Code

1.78.2

### Platform and Architecture

Windows 64 bit

### Additional Information

This is not necessarily an exhaustive list of all bad links.  Presumably `grep` would reveal at least all references to gnikit.github.io.

I started trying to understand why the extension reported it was unable to launch the language server.  I went to settings and set a full path, but it didn't help.  Looking for more info, I clicked on the documentation link associated with the next preference item, `Configure`, and discovered it was broken.

Then I tried the extension page, leading to the other discoveries.

**Comment by gnikit** [comment, marked as answer]
Hey @RossBoylan you should probably try the pre-release. The links are fixed there.
